tests/_test_msui/test_mscolab_operation.py

```diff
@@ -79,6 +81,10 @@ def test_send_message(self, qtbot):
         self._send_message(qtbot, "**test message**")
         with self.app.app_context():
             assert Message.query.filter_by(text='**test message**').count() == 2
+            message = Message.query.filter_by(text='**test message**').first()
+            result = get_message_dict(message)
+            assert result["message_type"] == MessageType.TEXT
+            assert datetime.datetime.fromisoformat(result["time"]).year == datetime.datetime.now().year
```

Comment:
```suggestion
            assert datetime.datetime.fromisoformat(result["time"]) == message.created_at
```

This will test that the full parsed datetime object matches the one recorded in the database, instead of just the year.

Reply:
will do that, I guess it throws an exception when the string does not match the format.

Reply:
It doesn't necessarily throw an exception, since most parts of the iso8601 format are optional. E.g. a date alone is parsed as that date at 00:00:
```python
>>> datetime.datetime.fromisoformat("2022-01-01")
datetime.datetime(2022, 1, 1, 0, 0)
```
A timezone doesn't have to specified either as also seen in that example, it is parsed to a naive datetime then.

That's why the year comparison is not enough.